plugin/src/main/java/io/jenkins/plugins/casc/Configurator.java

```diff
@@ -167,4 +171,19 @@ default CNode describe(T instance, ConfigurationContext context) throws Exceptio
         return mapping;
     }
 
+
+    @CheckForNull
+    default CNode describeStruct(T instance, ConfigurationContext context) {
+        return lookupConfig(context, instance.getClass()).map(configurator-> convertToNod(context, configurator, (Describable) instance)).getOrNull();
+    }
```

Comment:
@casz This here throws me a configurator exception because it cannot find `io.jenkins.plugins.casc.ConfiguratorException: Cannot find configurator for type class hudson.model.Hudson`.After close debugging it seems that the base type for which the look up is being done is not correct. But cannot point my finger at it . could be a silly mistake as well :P